README.md

```diff
@@ -2,32 +2,35 @@
 
 This handbook covers various aspects of work at Rapid River:
 
-* [Overview](#overview)
-  * [Mission](#mission)
-  * [Motto](#motto)
-  * [Founder](#founder)
-  * [Management staff](#management-staff)
-  * [Notable clients](#notable-clients)
-* [Working life](#working-life)
-  * [Working remotely](#working-remotely)
-  * [Hours](#hours)
-  * [Vacation policy](#vacation-policy)
-  * [On-call](#on-call)
-* [Communication](#communication)
-  * [Etiquette](#etiquette)
-  * [Email](#email)
-  * [Slack](#slack)
-* [Security](#security)
-  * [Physical space](#physical-space)
-  * [Computers](#computers)
-  * [Passwords](#passwords)
-  * [Network security](#network-security)
-  * [Other devices](#other-devices)
-  * [Encryption at rest](#encryption-at-rest)
-  * [Proprietary and client data](#proprietary-and-client-data)
-  * [Responses to security incidents](#responses-to-security-incidents)
-* [Feedback and improvements](#feedback-and-improvements)
-  * [Source of this document](#source-of-this-document)
+* [Rapid River Software Employee Handbook](#rapid-river-software-employee-handbook)
```

Comment:
I'm not sure that this extra level of nesting adds any value, since the top-level link just brings you to the document that you're already in. What use case are you imagining for the `#rapid-river-software-employee-handbook` anchor link?

Reply:
So this must have been an oversight. My code editor was pulling a fast one on me and I had to restart it. I must have gotten in there somehow.